gateway-sp-comms/src/error.rs

```diff
@@ -116,6 +116,8 @@ pub enum UpdateError {
     InvalidComponent,
     #[error("an image was not found")]
     ImageNotFound,
+    #[error("cannot update preferred boot image")]
```

Comment:
nitpicky, feel free to ignore: should this use the same string as the `UpdateError` variant?

Reply:
same string will be use in both.